rmshit.py

```diff
@@ -94,17 +96,31 @@ def yesno(question, default="n"):
         return True
     return False
 
+def format_size(size_in_bytes):
+    """Format file size in bytes to a human-readable string."""
+    if size_in_bytes <= 0:
+        return "0 bytes"
+
+    units = ['bytes', 'KB', 'MB', 'GB']
+    size = float(size_in_bytes)
+    unit_index = min(int((size_in_bytes.bit_length() - 1) // 10) , len(units) - 1)
+    size /= (1024 ** unit_index)
+
+    return f"{size:.2f} {units[unit_index]}"
```

Comment:
I think this will be more pretty (especially for the "bytes" unit):
```suggestion
    return f"{size:.4g} {units[unit_index]}"
```

Reply:
Thank you, I didn't know about this formatting option. It makes more sense since it won't return zeros at the and and always show 4 significant digits.